.dagger/test.go

```diff
@@ -159,6 +163,82 @@ func (t *Test) Specific(
 	)
 }
 
+// Run specific tests
+func (t *Test) Dump(
+	ctx context.Context,
+	// Only run these tests
+	// +optional
+	run string,
+	// Skip these tests
+	// +optional
+	skip string,
+	// +optional
+	// +default="./..."
+	pkg string,
+	// Abort test run on first failure
+	// +optional
+	failfast bool,
+	// How many tests to run in parallel - defaults to the number of CPUs
+	// +optional
+	parallel int,
+	// How long before timing out the test run
+	// +optional
+	timeout string,
+	// +optional
+	race bool,
+	// +default=1
+	// +optional
+	count int,
+	// Enable verbose output
+	// +optional
+	testVerbose bool,
+	// debug subroute to dump, like pprof/profile, pprof/heap, or requests
+	// +optional
+	// +default="pprof/heap"
+	route string,
```

Comment:
if possible we'll at some point want to be able to get e.g. both cpu profiles and heap profiles. Not sure how complicated that'd be right now